riski-frontend/public/fonts/fonts.css

```
@@ -0,0 +1,46 @@
+}
+
+@font-face {
+	font-family: 'Open Sans';
+	font-weight: 700;
+	font-style: normal;
+	src: url('/fonts/Open_Sans/OpenSans-Bold.ttf') format('truetype');
+}
+
+/* Font Awesome (PDF Annotation) */
+@font-face {
+	font-family: 'FontAwesome';
```

Stylelint (17.3.0)

[error] Unexpected quotes around "FontAwesome" (font-family-name-quotes)

(font-family-name-quotes)